blog/2023-06-14-paralus-at-platformcon-2023/index.md

```diff
@@ -0,0 +1,114 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/UbLnCGOv0Rk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+During [PlatformCon 2023](https://platformcon.com/), we were inundated with visitors reached us on Platform Engineering community [slack channel](https://platformengin-b0m7058.slack.com/join/shared_invite/zt-1vsqjlb51-SiDy9cp2EBGffIUj5_g2mA#/shared-invite/email), who were keen to learn more about our project. We had some quick demos on how to use the project, along with some tasty cookies and cool stickers to give away. The visitors were curious about the project and its potential, and some of them even reached out for a dedicated demo of the tool post-platformcon.   
+
+Big topic during PlatformCon was the concept of platform engineering. As software gets more complex and Kubernetes enters the enterprise, the idea of Kubernetes being a “platform for platforms” is becoming more and more ubiquitous. 
+
+A bit about “Platform engineering” is a discipline focused on building and maintaining the underlying infrastructure and tools that enable developers to build, deploy, and run software applications efficiently. It involves creating a stable and scalable platform that provides a foundation for software development, deployment, and operations.
+
+**[Six Guiding Principles For Platform Teams](https://www.forbes.com/sites/forbestechcouncil/2023/02/23/how-to-empower-modern-kubernetes-management-with-a-platform-team-model/?sh=1690484c52c7)**: 
+- Consistent Automation
+- Security Preparedness
+- Centralized Visibility
+- Governance
+- Kubernetes Distribution Flexibility
+- Oriented To Self-Service
+
+*In this blog we will be referring to more on **Kubernetes access management** or better to call it **Security Preparedness:** *
+
+With the right access to the right roles, separation of duties becomes easier to implement and enforce, and it becomes easier for the platform and security teams to protect against unauthorized use.  
+
+## Why is to Secure Access to k8s Important? 
+
+The inability to secure K8s infrastructure is a growing problem for organizations. In May 2022, a non-profit security organization named [The Shadowserver Foundation](https://www.shadowserver.org/news/over-380-000-open-kubernetes-api-servers/) scanned more than 450,000 systems hosting K8s and found more than 380,000 (84 percent) of these systems were accessible via the Internet, potentially providing an opening into a corporate network. In fact, the data shows that the majority of K8s API servers are found in the United States (nearly 53 percent).
```

Comment:
Let us be consistent with using Kubernetes and not K8s. Please update that across the blog post.